### PR TITLE
update doc with managed identities

### PR DESCRIPTION
**Pull request by ilappe**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-1008

## Description
added a new section describe the Azure Managed Identities

## Screenshots
image
image

### CODE REVIEW

**Comment by ilappe** [comment]
@yucohen 
please review the table added
feel free to edit/add/delete

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 1]
```diff
@@ -101,6 +102,21 @@ To configure a Microsoft integration that uses this authorization flow with a se
...
+
+Some of the Cortex XSOAR-Microsoft integrations use the [Azure Managed Identities Authentication](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview).
+
+Follow one of these steps for authentication based on Azure Managed Identities:
+
+- ##### To use System Assigned Managed Identity
+   - Select **Azure Managed Identities** from the **Authentication Type** drop down or select the **Use Azure Managed Identities** checkbox and leave the **Azure Managed Identities Client ID** field empty.
+
+- ##### To use User Assigned Managed Identity
+   1. Go to the [Azure Portal](https://portal.azure.com/) -> **Managed Identities**
```
Comment (julieschwartz18) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:116]:
```suggestion
   1. Go to [Azure Portal](https://portal.azure.com/) -> **Managed Identities**.
```

**Comment by bakatzir** [comment]
> @yucohen @bakatzir please review the table added feel free to edit/add/delete

Thanks israel. a few Qs I would love clarification above/under the table:
1. There is a XSOAR application column, with differentiation between Oproxy and Device, but not at the article: https://xsoar-pan-dev--pull-request-1288-fiwkzbr5.web.app/docs/reference/articles/microsoft-integrations---authentication#cortex-xsoar-application. Am I missing something?

2. Client credentials. what is the difference in between Yes, Yes with certificate, and why API that require user impersonation does not aloow it? https://xsoar-pan-dev--pull-request-1288-fiwkzbr5.web.app/docs/reference/articles/microsoft-integrations---authentication#client-credentials

3. Same for the 2 Nos in Auth code column.

**Comment by yucohen** [comment]
@bakatzir 
>There is a XSOAR application column, with differentiation between Oproxy and Device, but not at the article:

There is no difference for the customer using the application - in both ways the application is on XSOAR Azure portal. I think we can remove this tho.

> Client credentials. what is the difference in between Yes, Yes with certificate

Yes - the client credentials flow requires an app secret OR certificate. We will improve this documentation. 

> and why API that require user impersonation does not allow it

Those API's do not support the client credentials flows  - it requires a user involved in the authentication. Do you think we need to explain? Or a simple No will do?

**Review by yucohen — APPROVED** [review]
Amazing!

**Pull request merged**